### PR TITLE
Updates the contributor policy to reflect the new policy in place

### PR DESCRIPTION
**Pull request by deathride58**
Title. This also elaborates on how approval for a PR under the new "requires prior approval" section may be granted.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -414,10 +414,17 @@ Regarding sprites & sounds, you must credit the artist and possibly the codebase
...
 
 Just because something isn't on this list doesn't mean that it's acceptable. Use common sense above all else.
 
+## Content requiring prior approval
+Certain types of changes may require prior approval from maintainers. This currently includes:
+
+* Code adding, removing, or updating the availability of alien races/species/human mutants. This includes pull requests attempting to add or remove features from said races/species/mutants.
+* Code adding, removing, or modifying the functionality of adult-oriented features (such as, but not limited to: vore, genitals, MKUltra, and more). This also includes pull requests attempting to add or remove these features outright.
+
+The above content requires approval from at least half of the formal maintainer team prior to PR creation. Seeking approval must be done via a @Maintainer ping in a relevant development/code or staff channel on the Discord, otherwise it will be considered insufficient. If a PR contains any of the above content, but the creator does not have sufficient approval prior to the PR's creation, then the PR may be closed by any maintainer, at any time, for any reason.
```
Comment (silicons) [inline .github/CONTRIBUTING.md]: this should only apply t oadult features, and not species.

**Comment by timothyteakettle** [comment]
wow you can make a change to the policy so you can reference your already existing ability to ban prs because you don't like them
thanks this definitely will solve
oh yeah it'll solve nothing thanks

**Comment by Ludox235** [comment]
bhjin making literal salt pr because he doesn't want mechanical erp to be deleted

**Comment by CameronWoof** [comment]
Maintainers already have the ability to deny PRs from getting merged. Why do they need the ability to deny PRs from getting made? Was absolute power not enough?

**Comment by nik707** [comment]
god imagine being mad that we just want to save everyone time

**Review by nik707 — APPROVED** [review]

**Comment by timothyteakettle** [comment]
remove the part about species so i don't have to ping every maintainer every 5 minutes to notify you that i am making yet another change to species features thank you

**Pull request merged**